packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -259,7 +266,8 @@ describe('OutgoingPaymentService', (): void => {
     ).resolves.toBeUndefined()
 
     incomingPayment = await createIncomingPayment(deps, {
-      walletAddressId: receiverWalletAddress.id
+      walletAddressId: receiverWalletAddress.id,
+      client: client
```

Comment:
I think we can remove this since we are not asserting anything about the incoming payment in this test file